### PR TITLE
Prevent taglib from crashing kodi

### PR DESCRIPTION
**Pull request by razzeee**
Backport of https://github.com/xbmc/xbmc/pull/8524

@fritsch

### CODE REVIEW

**Comment by Razzeee** [comment]
jenkins build this please

File: xbmc/music/tags/TagLoaderTagLib.cpp [hunk 1]
```diff
@@ -832,54 +832,61 @@ bool CTagLoaderTagLib::Load(const std::string& strFileName, CMusicInfoTag& tag,
...
+  {
+    CLog::Log(LOGERROR, "Taglib exception: %s", ex.what());
+  }
 
   if (!file || !file->isOpen())
   {
     delete file;
     delete stream;
-    CLog::Log(LOGDEBUG, "file could not be opened for tag reading");
+    CLog::Log(LOGDEBUG, "file %s could not be opened for tag reading", strFileName.c_str());
```
Comment (ace20022) [inline xbmc/music/tags/TagLoaderTagLib.cpp]: Redacted filename?

Comment (Razzeee) [inline xbmc/music/tags/TagLoaderTagLib.cpp]: Not sure what can be in a filename, that we could redact, without making this obsolete.
A path might have a password, but a file name shouldn't include sensible data? Or am I wrong?

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: xbmc/music/tags/TagLoaderTagLib.cpp (hunk 1 above)
Comment (ace20022) [inline xbmc/music/tags/TagLoaderTagLib.cpp]: Right, have not checked what it is exactly, sorry for the noise.

Comment (Razzeee) [inline xbmc/music/tags/TagLoaderTagLib.cpp]: No problem, thanks for the sanity check :)

**Pull request merged**